indico/modules/designer/client/js/index.js

```diff
@@ -84,10 +84,12 @@ import {$T} from 'indico/utils/i18n';
       italic: false,
       text_align: 'center',
       color: 'black',
+      background_color: null,
       font_size: '15pt',
       width: isImage(type) ? 150 : 400,
       height: isImage(type) ? 150 : null,
       text: $T('Fixed text'),
+      zIndex: itemIdCounter + 100,
```

Comment:
Why 100 and not 10 like before?

Reply:
This is because background images have a z-index of 5 and since moving items forward/backward adds +/-1 to the z-index value, it could easily become less than 5, placing the item behind the background image. Which is why I used 100 instead of 10.

Reply:
Yeah, I get it. However, it is still possible that, after a few runs this gets again to under 5. Could we maybe have a function which "normalizes" the z-indexes after each operation? E.g. sort them in growing order, assign them z-indices according to the sequence of natural numbers (1, 2, 3, 4...) and then add 9 to all? This would keep them to a minimum. It would also hold more consistent results when you move things around.

Reply:
Yeah, this would work